frontend/src/components/hook-form/RHFTextField.tsx

```diff
@@ -20,6 +20,7 @@ export default function RHFTextField({ name, ...other }: Props) {
       control={control}
       render={({ field: { ref, ...field }, fieldState: { error } }) => (
         // Note: Have to take out the ref for ref to passed succesfully
+        // TODO: check if this field is required and automatically toggle `required` prop
```

Comment:
Elaborate?

Reply:
MUI TextField has a required prop: If true, put an asterisk to indicate beside the label. I am thinking whether we are able to know if a particular field is required from the form context so we can just pass it to the MUI TextField without manually insert the prop